tests/bud/buildkit-mount/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+RUN mkdir /test
+# use option z if selinux is enabled
+RUN --mount=type=bind,source=.,target=/test,z cat /test/input_file
```

Comment:
Please also have a RUN with no `source` speciified, and with the source being a subdirectory under the context directory, given both with and without a leading "/".

Reply:
sure will add few more tests.